libs/xc/xenctrl.ml

```diff
@@ -310,6 +310,12 @@ external get_featureset : handle -> featureset_index -> int64 array = "mock1"
 external upgrade_oldstyle_featuremask: handle -> int64 array -> bool -> int64 array = "mock1"
 external oldstyle_featuremask: handle -> int64 array = "mock1"
 
+(* New MSR external functions *)
+external xc_get_msr_arch_caps : handle -> int64 = "mock1"
+external xc_cpu_policy_get_system: handle -> int -> int64 array = "mock1"
```

Comment:
I don't think it really matters if we never run any code but `mock1` is for a C function that takes one OCaml value, and `mock2` for two values and so on. So this should be `mock2`. Likewise below.

Reply:
Obviously there are several other wrong bindings here already.

Reply:
Ah, I did know this, but must have forgotten to update them as I changed the parameters to the functions. I'm assuming that the handle doesn't count as a parameter?

Reply:
I would expect the C code to receive the handle as well, so it would count.

Reply:
Hmm, ok I'll update them all to include this as a parameter - I think I ascertained my misconception from this file, so it might well be that this file is more wrong than right....